awx/playbooks/heartbeat_isolated.yml

```diff
@@ -1,7 +1,7 @@
 ---
 - name: Periodic background status check of isolated instances.
   hosts: all
-  gather_facts: false
+  gather_facts: true
```

Comment:
This could make health checks for large clusters of isolated nodes notably slower.